### DISCUSSION TITLE
Option to disable automatic merging of recent changes into drafts

### DISCUSSION THREAD

**Discussion by lenvanessen** [craftcms/cms#9911] [Ideas] [answered]
Can we have an option that disables the behavior added in Craft 3.7. We don't need nor want changes to be automatically merged into drafts?

**Comment by alexanderloewe** [comment]
We like to have this option as well.

**Comment by lenvanessen** [comment, marked as answer]
duplicate of https://github.com/craftcms/cms/discussions/9053
@brandonkelly will you accept a pr fo this?

**Reply by brandonkelly** [reply]
Will follow up on #9053.
